### ISSUE TITLE
Blog does not pass W3C-validation

### ISSUE THREAD

**Issue by mattiasaxell** [openknowledge-archive/wordpress-theme-with-bp#36]
Hi!

Got feedback from a blog visitor that our blog does not pass W3C validation.

See here: https://validator.w3.org/feed/check.cgi?url=http%3A%2F%2Fse.okfn.org%2Fcategory%2Fproject%2Ffragastaten%2Ffeed%2F

As you can see this is a customized category feed which needs to be functioning properly as it also is mirrored at https://fragastaten.se/blog

Is this an issue with the wordpress-theme? If it is something at our end please let me know.

**Comment by morchickit** [comment]
@gocemitevski have you looked at this?

**Comment by gocemitevski** [comment]
@morchickit not yet. Will have a look as soon as possible.

**Comment by gocemitevski** [comment]
@mattiasaxell this is definitely not related to the theme. It is related to the content of a specific post. There were some invalid unicode character infront of `<!-- kommunen? -->` on line 156 and those are causing the validation error. I removed them and the error is now fixed.

Depending on how content is cached on the website, these changes will be visible immediately or once the cache expires.

cc @morchickit
